scripts/preprocessing/_01_maxfilter.py

```diff
@@ -24,25 +24,85 @@
 
 import numpy as np
 import mne
-from mne_bids import BIDSPath
+from mne_bids import BIDSPath, read_raw_bids
 
 import config
 from config import (gen_log_kwargs, on_error, failsafe_run,
-                    import_experimental_data, import_er_data, import_rest_data,
-                    get_reference_run_params)
+                    import_experimental_data, import_er_data, import_rest_data)
 from config import parallel_func
 
 
 logger = logging.getLogger('mne-bids-pipeline')
 
 
-@failsafe_run(on_error=on_error, script_path=__file__)
-def run_maxwell_filter(*, cfg, subject, session=None, run=None):
+def _get_reference_run_path(
```

Comment:
what's with this function, it doesn't seem to be used anywhere?

Reply:
Indeed `git grep` says it's no longer used, feel free to push a commit to delete it!